lsp-mode.el

```diff
@@ -1316,13 +1316,41 @@ depending on it."
          :is-incomplete (seq-some
                         #'lsp:completion-list-is-incomplete
                         results)
-         :items (apply 'append (--map (append (if (lsp-completion-list? it)
-                                                  (lsp:completion-list-items it)
-                                                it)
-                                              nil)
-                                      results))))
-       (_ (apply 'append (seq-map (lambda (it) (if (seqp it) it (list it)))
-                                  results)))))))
+         :items (mapcan (lambda (it) (append (if (lsp-completion-list? it)
```

Comment:
I don't think `mapcan` is available on older Emacsen, so it should be `cl-mapcan` to accommodate.